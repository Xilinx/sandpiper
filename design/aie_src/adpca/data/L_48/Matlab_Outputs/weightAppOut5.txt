1.807384491e-01 2.292513609e+00
3.221992970e+00 1.163692236e+00
-6.312223911e+00 -1.520336151e+00
