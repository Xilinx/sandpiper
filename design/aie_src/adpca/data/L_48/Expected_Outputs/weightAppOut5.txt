1.806627512e-01 2.292472601e+00 
3.221935272e+00 1.163707256e+00 
-6.312024593e+00 -1.520206451e+00 
